### PR TITLE
uploading ftw.png

### PR DESCRIPTION
**Pull request by xxedocxx**

### CODE REVIEW

**Comment by mhuggins** [comment]
This looks a little pixelated or something.  Should we use the 512x512 version from their homepage?  https://www.ftwcoin.io/favicons/android-chrome-512x512.png

File: tokenList.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
         "totalSupply": 342015750
       }
     },
-    "image": "https://rawgit.com/CityOfZion/neo-tokens/master/assets/png/asa.png"
+    "image": "https://rawgit.com/undefined/neo-tokens/master/assets/png/asa.png"
```
Comment (mhuggins) [inline tokenList.json]: Looks like a bad merge happened here.  If it's easier, feel free to open a new PR with the file updated.

Comment (xxedocxx) [reply tokenList.json]: Got it.

**Comment by xxedocxx** [comment]
I did

**Pull request closed without merging**